components/micro-gateway-core/src/main/ballerina/src/gateway/filters/pre_authn_filter.bal

```diff
@@ -0,0 +1,157 @@
+// Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License year?
```suggestion
// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
These are not new files, I have renamed and moved them